pom.xml

```diff
@@ -212,6 +213,13 @@
         <version>${version.org.assertj_assertj-core}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
it's a matter of preference. I'd prefer to have a single place for `dependencyManagement` in my projects